Expense/Expense.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const Expense = new mongoose.Schema({
+    amount: Number,
+    description: String,
+    budget:[{
+        type: ObjectId,
+        ref: "Budget"
+    }],
+    category:[{
```

Comment:
Same here, there are many Expenses, but only one Category for each, so this property shouldn't be in an array.